diagnostic.md

```diff
@@ -48,11 +54,11 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    model: function(params){};
```

Comment:
You are right that you need to use params.  You would want to use `params.movie_id`